client/Dockerfile

```diff
@@ -180,3 +182,22 @@ RUN if [ "$BUILD_QT" != "1" ] ; then \
     cd .. && \
     rm -rf qtkeychain \
     ; fi
+
+# Setup sonar-scanner
+ENV SONAR_SCANNER_VERSION 4.6.1.2450
+ENV SONAR_SERVER_URL "https://sonarcloud.io"
+ENV SONAR_SCANNER_DOWNLOAD_URL https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-${SONAR_SCANNER_VERSION}-linux.zip
+
+RUN mkdir -p /opt/sonar && \
+    curl -sSLo /opt/sonar/sonar-scanner.zip ${SONAR_SCANNER_DOWNLOAD_URL} && \
+    unzip -o /opt/sonar/sonar-scanner.zip -d /opt/sonar/
+
+ENV PATH=/opt/sonar/sonar-scanner-${SONAR_SCANNER_VERSION}-linux/bin:${PATH}
+
+# Setup build-wrapper
+ENV BUILD_WRAPPER_DOWNLOAD_URL ${SONAR_SERVER_URL}/static/cpp/build-wrapper-linux-x86.zip
```

Comment:
why is it `x86` and not `amd64` or `x86_64` ?

Reply:
Good question, but that is from the official guide, and I don't see that other versions exist.

Reply:
OK, can you add a pointer to the doc here for later reference ?